spring-cloud-alibaba-starters/spring-cloud-circuitbreaker-sentinel/src/test/java/com/alibaba/cloud/circuitbreaker/sentinel/ReactiveSentinelCircuitBreakerIntegrationTest.java

```diff
@@ -62,95 +64,69 @@ public class ReactiveSentinelCircuitBreakerIntegrationTest {
 	@Autowired
 	private ReactiveSentinelCircuitBreakerIntegrationTest.Application.DemoControllerService service;
 
+	private final AtomicInteger idx = new AtomicInteger(0);
+
 	@Before
 	public void setup() {
 		service.setPort(port);
 	}
 
+	@Repeat(100)
 	@Test
 	public void test() throws Exception {
-		StepVerifier.create(service.normal()).expectNext("normal").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-
-		// Then in the next 5s, the fallback method should be called.
-		for (int i = 0; i < 5; i++) {
-			StepVerifier.create(service.slow()).expectNext("fallback").verifyComplete();
-			Thread.sleep(900);
+		int idx = this.idx.getAndIncrement();
```

Comment:
Please add some comment to explain your logic here, it's not easy to understand the logic in short time.

Reply:
This PR is just a test